src/plugins/visualizations/server/saved_objects/visualization_migrations.ts

```diff
@@ -780,6 +787,6 @@ export const visualizationSavedObjectTypeMigrations = {
   '7.3.1': flow(migrateFiltersAggQueryStringQueries),
   '7.4.2': flow(transformSplitFiltersStringToQueryObject),
   '7.7.0': flow(migrateOperatorKeyTypo, migrateSplitByChartRow),
-  '7.8.0': flow(migrateTsvbDefaultColorPalettes),
+  '7.8.0': flow(migrateMatchAllQuery, migrateTsvbDefaultColorPalettes),
```

Comment:
Because there's an existing 7.8.0 migration, anyone who has already upgraded to 7.8.0 and then upgrades to 7.8.2 won't get this `migrateMatchAllQuery` applied. For consistency, let's mark all the `migrateMatchAllQuery` migrations as v7.8.2 (visualisations, search and dashboard)

Reply:
I agree, `7.8.2` should be a target version. Thank you